src/test/java/org/jivesoftware/util/StringUtilsTest.java

```diff
@@ -5,10 +5,18 @@
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.fail;
 
+import java.util.Locale;
+
+import org.junit.Before;
 import org.junit.Test;
 
 public class StringUtilsTest {
 	
+	@Before
```

Comment:
Install Editor Config and don't have weird indentation like this any more! :)

Reply:
Done.